src/test/java/com/thebois/models/world/structures/HouseTests.java

```diff
@@ -183,4 +187,37 @@ public void whenStructureHasReceivedItemsNeededItemsAreEqualToExpected(
         assertThat(neededItems.containsAll(expectedNeededItems)).isTrue();
     }
 
+    private static Stream<Arguments> whenItemsHaveBeenDeliveredStructureReturnsCorrectIsCompleteSource() {
```

Comment:
style

I'm not a fan of this way of naming arguments. Couldn't it be called `getItemAmountsAndExpectedCompleteness` or something?

Reply:
I named them like this because I noticed a lot of these "testArguments" methods would end up with the same names as each other even though they didn't have the same parameters or were for the same tests.

Reply:
Do you have any examples? It sounds like they shouldn't have the same names if they don't have the same parameters or are not even used for the same tests.

Reply:
Maybe I was just naming them wrong, guess it works